### ISSUE TITLE
 Symbol's function difinition is void: doom|transient-hook--load-projectile

### ISSUE THREAD

**Issue by chengyi** [hlissner/doom-emacs#1278]
### Observed behavior

Start Emacs, the dashboard show, press `SPC h f`, then press `ctrl-g`, then press `SPC h f` again.
the helm related function work abnormal, emacs says: Symbol's function difinition is void: doom|transient-hook--load-projectile

### Expected behavior

just work perfect

### Steps to reproduce

1.  press `SPC h f`
2.  press `ctrl-g`
3.  press `SPC h f`

### System information

<details>
<summary>Click to expand</summary>

```
- OS: gnu/linux (x86_64-pc-linux-gnu)
- Emacs: 26.1 (7月 05, 2018)
- Doom: 2.0.9 (develop 1638d09a198f970d44966c0265abcb81f332105d)
- Graphic display: t (daemon: nil)
- System features: XPM JPEG TIFF GIF PNG RSVG IMAGEMAGICK SOUND GPM DBUS GSETTINGS NOTIFY ACL GNUTLS LIBXML2 FREETYPE M17N_FLT LIBOTF XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 MODULES THREADS LIBSYSTEMD LCMS2
- Details:
  ```elisp
  elc count: 0
  uname -a:  Linux prow-dev 4.19.28-1-MANJARO #1 SMP PREEMPT Sun Mar 10 08:32:42 UTC 2019 x86_64 GNU/Linux
  modules:   (:feature eval (evil +everywhere) file-templates (lookup +docsets) snippets workspaces :completion company helm :ui doom doom-dashboard doom-quit evil-goggles fci hl-todo modeline nav-flash treemacs (popup +all +defaults) vc-gutter vi-tilde-fringe window-select :editor fold multiple-cursors rotate-text :emacs dired electric imenu term vc :tools magit :lang cc data emacs-lisp java lua markdown (org +attach +babel +capture +export +present) python rust (sh +zsh) :config default)
  packages:  n/a
  exec-path: (/usr/local/bin /usr/bin /bin /usr/local/sbin /opt/android-ndk /usr/lib/jvm/default/bin /usr/bin/site_perl /usr/bin/vendor_perl /usr/bin/core_perl /usr/lib/emacs/26.1/x86_64-pc-linux-gnu)
  ```
```
</details>

**Comment by hlissner** [comment]
This should be fixed as of d7e9b47. Let me know if that isn't the case and I will reopen this issue. Thank you for bringing it my attention!

**Comment by chengyi** [comment]
@hlissner it works.:+1:
